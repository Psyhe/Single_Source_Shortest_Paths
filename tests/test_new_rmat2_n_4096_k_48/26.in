4096 2226 2310
2226 48 59
2226 187 175
2226 2098 172
2226 2102 244
2226 2082 218
2226 2290 17
2226 145 222
2226 2746 184
2226 526 52
2226 2194 236
2226 2918 3
2226 2482 199
2226 2072 224
2226 2065 192
2226 2098 100
2226 1412 237
2226 3258 120
2226 546 51
2226 2210 51
2226 3250 28
2226 2202 18
2227 929 175
2227 1214 161
2227 2176 123
2227 547 28
2227 2208 17
2227 59 134
2227 2209 1
2227 2291 8
2227 59 29
2227 19 0
2227 3123 76
2227 54 0
2227 439 2
2227 2195 43
2228 2748 236
2228 2236 195
2228 2164 189
2228 2544 223
2228 3956 106
2228 2484 28
2228 149 85
2228 2100 136
2228 3125 215
2228 2225 56
2228 2356 112
2228 2237 87
2228 2232 101
2228 2105 103
2228 4016 86
2228 2231 240
2228 180 184
2228 2224 204
2228 1172 122
2228 2230 3
2228 176 11
2228 2460 124
2229 2096 158
2229 3735 120
2229 2485 90
2229 3236 118
2229 145 122
2229 2484 235
2229 3092 161
2229 177 94
2229 2196 204
2229 2433 19
2229 1205 191
2229 2237 1
2229 1109 109
2229 3253 72
2229 181 139
2229 2101 107
2229 2741 158
2230 148 115
2230 2228 3
2230 4030 59
2230 3762 252
2230 1074 167
2230 2356 99
2230 2801 134
2230 165 149
2230 246 124
2230 3314 51
2230 246 210
2230 3126 246
2230 182 246
2231 2215 89
2231 190 141
2231 3714 133
2231 2183 227
2231 3127 167
2231 20 101
2231 2295 162
2231 2228 240
2232 520 24
2232 2228 101
2232 2808 193
2232 2488 188
2232 304 168
2232 2616 243
2232 2096 16
2232 2194 39
2232 172 67
2232 424 5
2232 2194 149
2232 3260 253
2232 2201 198
2232 2224 186
2232 2048 111
2232 41 142
2232 176 160
2232 188 223
2232 2104 1
2232 2168 234
2232 2457 44
2232 2200 215
2232 649 31
2233 3209 199
2233 2235 34
2233 2594 158
2233 2201 86
2233 3312 214
2233 2216 224
2233 2682 217
2233 2248 84
2233 2296 166
2233 72 19
2233 2201 104
2233 2215 193
2233 2274 46
2233 2072 207
2233 2495 174
2233 2737 85
2233 2104 95
2234 2808 229
2234 186 85
2234 153 19
2234 3130 216
2234 3219 105
2234 146 211
2234 1210 27
2234 2736 222
2234 2442 206
2234 154 242
2234 2235 212
2235 2234 212
2235 2203 70
2235 2233 34
2235 2744 110
2235 1211 54
2235 2175 223
2235 3082 16
2235 2132 39
2235 2151 82
2235 2139 76
2235 2207 214
2236 413 81
2236 2221 125
2236 3260 149
2236 2092 28
2236 2150 92
2236 2196 43
2236 172 9
2236 1201 103
2236 2524 183
2236 2228 195
2236 1256 19
2237 3189 66
2237 3116 76
2237 2209 31
2237 139 224
2237 2229 1
2237 2076 119
2237 2196 240
2237 2228 87
2238 2462 152
2238 3212 173
2238 2196 23
2238 2109 168
2238 2214 239
2238 3130 164
2239 2295 5
2239 2207 165
2239 251 198
2239 2170 117
2240 2241 247
2240 196 43
2240 3328 21
2240 2241 48
2240 2368 75
2240 192 79
2240 2113 18
2240 2694 66
2240 320 85
2240 417 209
2240 2296 28
2240 2280 227
2240 1250 251
2240 323 62
2240 192 96
2240 2241 212
2240 452 54
2240 2176 173
2240 2752 241
2240 3200 147
2240 2115 33
2240 2112 217
2240 2160 28
2240 2241 127
2240 1098 145
2240 192 207
2240 322 251
2240 2136 211
2240 68 153
2240 3264 51
2240 704 252
2240 2772 28
2240 3786 89
2240 2498 108
2240 2752 194
2240 192 1
2240 2147 254
2241 2240 247
2241 2240 48
2241 193 82
2241 2245 6
2241 2113 237
2241 2049 188
2241 194 28
2241 3281 127
2241 2240 212
2241 2311 158
2241 3526 239
2241 208 113
2241 2425 61
2241 2113 27
2241 2240 127
2241 3265 249
2241 2261 88
2241 200 96
2241 3267 27
2241 2243 55
2241 2281 83
2241 5 125
2241 74 141
2241 2369 61
2241 3264 145
2241 2157 116
2241 2057 64
2241 2257 52
2241 960 192
2242 2178 132
2242 2514 232
2242 3714 41
2242 197 122
2242 199 42
2242 2952 141
2242 195 235
2242 2051 100
2242 710 17
2242 3074 156
2242 194 193
2242 2112 212
2242 2530 213
2242 2442 155
2242 320 135
2242 203 113
2242 2752 32
2242 3584 35
2242 233 162
2242 258 209
2242 2252 193
2242 210 146
2242 2218 107
2242 1010 50
2242 658 235
2242 2670 101
2242 128 56
2242 2250 198
2242 3720 127
2242 582 151
2242 2114 240
2242 198 133
2242 3776 121
2242 706 226
2242 2275 215
2242 2370 13
2243 2259 162
2243 726 102
2243 193 131
2243 3219 143
2243 2802 180
2243 130 250
2243 2674 81
2243 2771 242
2243 195 44
2243 2259 228
2243 227 231
2243 2497 0
2243 3139 179
2243 2752 123
2243 3154 170
2243 2246 24
2243 129 238
2243 2115 131
2243 579 251
2243 67 174
2243 2506 243
2243 2241 55
2243 2755 150
2243 3777 129
2243 2698 19
2243 2258 162
2244 725 48
2244 2300 102
2244 2182 162
2244 228 137
2244 2504 222
2244 2245 229
2244 2692 131
2244 2310 112
2244 2252 107
2244 2436 174
2244 2112 111
2244 2245 136
2244 388 56
2244 452 179
2244 2249 52
2244 196 175
2244 2246 116
2244 2513 220
2244 3844 73
2244 2190 96
2244 1188 46
2244 1121 111
2244 1240 104
2244 2692 246
2244 2272 8
2244 2246 210
2244 644 65
2244 3521 222
2244 3148 131
2244 3268 142
2244 2286 207
2244 2284 141
2244 580 146
2244 2188 120
2244 2753 70
2244 546 229
2244 260 3
2244 2502 32
2244 2124 30
2244 2048 19
2245 2241 6
2245 2133 140
2245 2246 5
2245 2116 83
2245 2052 237
2245 231 74
2245 3652 98
2245 2509 83
2245 3781 145
2245 2247 71
2245 3269 141
2245 2196 200
2245 2244 229
2245 2244 136
2245 2247 142
2246 452 129
2246 2243 24
2246 2132 193
2246 704 147
2246 2245 5
2246 2085 152
2246 2479 86
2246 2497 210
2246 2822 238
2246 2262 79
2246 2262 173
2246 2276 83
2246 3202 175
2246 3266 172
2246 2528 244
2246 68 168
2246 3279 159
2246 2244 116
2246 197 75
2246 1216 138
2246 2244 210
2247 2245 142
2247 2502 107
2247 2245 71
2247 651 121
2247 199 56
2247 2183 19
2247 2179 116
2247 2205 28
2248 2421 136
2248 2185 97
2248 2536 112
2248 2250 145
2248 2440 163
2248 2186 124
2248 2233 84
2248 1156 181
2248 1225 241
2248 2184 253
2248 2116 203
2248 2112 9
2248 2760 100
2248 3017 33
2248 2312 77
2248 32 116
2248 2252 6
2248 728 2
2248 3272 211
2248 3272 138
2248 1260 144
2248 2074 121
2248 2506 180
2248 192 226
2248 2440 182
2248 3154 143
2248 138 226
2249 2713 248
2249 2521 82
2249 3273 47
2249 2765 12
2249 2780 127
2249 217 140
2249 2269 98
2249 456 207
2249 2244 52
2249 729 250
2249 3307 82
2249 2121 110
2249 201 18
2249 2558 129
2249 2189 17
2249 2569 22
2249 2280 249
2249 137 124
2249 1223 144
2249 2187 142
2249 2505 197
2249 2120 177
2249 3401 77
2250 2216 211
2250 74 242
2250 154 118
2250 3291 170
2250 222 55
2250 458 204
2250 2762 11
2250 1754 238
2250 2154 42
2250 3036 75
2250 3288 178
2250 452 150
2250 2248 145
2250 2251 69
2250 2242 198
2250 2120 131
2250 2754 73
2250 2778 80
2250 2264 237
2251 2058 72
2251 2255 213
2251 2699 59
2251 2506 235
2251 2122 91
2251 2250 69
2251 458 14
2251 203 6
2251 2539 238
2251 2753 110
2251 2255 126
2251 2257 233
2252 236 191
2252 2176 182
2252 2244 107
2252 204 200
2252 3212 137
2252 3274 35
2252 2108 172
2252 2092 41
2252 2180 22
2252 108 113
2252 2804 22
2252 69 33
2252 2242 193
2252 1230 179
2252 2248 6
2252 204 80
2252 2380 44
2252 328 133
2252 1480 32
2252 2204 61
2252 2284 75
2253 3212 99
2253 3077 200
2253 2061 252
2253 3245 248
2253 2255 115
2253 2765 244
2253 2179 190
2253 2061 30
2253 3145 100
2253 2212 74
2253 2156 211
2253 2189 204
2254 2730 188
2254 2190 7
2254 2446 198
2254 3210 153
2254 2148 149
2254 462 8
2254 2206 187
2254 430 23
2254 2468 159
2254 198 191
2254 202 238
2254 3467 77
2254 234 112
2254 223 201
2255 2758 11
2255 223 173
2255 2251 213
2255 2767 86
2255 2213 129
2255 2223 142
2255 2273 75
2255 2253 115
2255 2251 126
2255 2655 46
2255 2598 1
2256 2132 222
2256 16 100
2256 2196 89
2256 1728 170
2256 2132 125
2256 2131 40
2256 2516 132
2256 672 7
2256 592 131
2256 211 151
2256 2064 62
2256 40 90
2256 832 46
2256 208 252
2256 224 204
2256 240 0
2256 472 179
2256 2192 98
2256 3218 207
2256 3284 39
2257 337 134
2257 2073 126
2257 2202 12
2257 3305 102
2257 2192 254
2257 2778 51
2257 176 26
2257 2260 193
2257 2961 232
2257 74 38
2257 2251 233
2257 2481 155
2257 2264 172
2257 499 153
2257 725 84
2257 757 255
2257 3312 162
2257 2129 18
2257 2241 52
2257 1361 126
2258 2690 69
2258 2291 188
2258 2288 179
2258 210 110
2258 1168 61
2258 2514 140
2258 2450 143
2258 194 131
2258 2 124
2258 2243 162
2259 2755 50
2259 2171 239
2259 2195 199
2259 243 232
2259 2289 18
2259 2265 201
2259 243 160
2259 83 69
2259 2243 162
2259 403 103
2259 1175 240
2259 2243 228
2260 212 193
2260 2261 156
2260 2968 2
2260 212 226
2260 2496 60
2260 2752 138
2260 724 70
2260 372 210
2260 228 4
2260 20 134
2260 2265 89
2260 4061 13
2260 2452 250
2260 2257 193
2260 2550 148
2260 3284 219
2260 2807 238
2261 2241 88
2261 2769 203
2261 3221 95
2261 2181 62
2261 3225 3
2261 2271 77
2261 724 113
2261 2260 156
2261 1526 250
2261 2387 114
2261 220 74
2261 2065 222
2261 2077 220
2261 2263 0
2261 1012 153
2261 221 141
2262 2132 210
2262 2290 188
2262 2263 136
2262 2150 70
2262 2246 79
2262 2576 207
2262 2246 173
2262 726 229
2262 2270 198
2262 2142 164
2262 2198 251
2262 3162 188
2262 158 145
2263 2261 0
2263 2262 136
2263 3797 230
2263 3286 239
2263 151 75
2263 215 191
2263 223 244
2263 727 98
2264 2137 45
2264 220 239
2264 2265 7
2264 265 7
2264 2064 7
2264 2352 196
2264 3568 203
2264 2266 133
2264 992 168
2264 2520 248
2264 3092 76
2264 2268 172
2264 2250 237
2264 2257 172
2264 2268 112
2264 2141 128
2264 216 211
2264 2120 170
2265 477 242
2265 2260 89
2265 1241 181
2265 2264 7
2265 2793 29
2265 2393 136
2265 2259 201
2265 3153 89
2266 2112 43
2266 2267 207
2266 194 156
2266 2264 133
2266 2650 27
2266 490 243
2266 2169 77
2266 3226 18
2267 91 15
2267 2266 207
2267 2803 116
2267 1242 187
2267 217 104
2267 2139 228
2268 3146 70
2268 3277 163
2268 2264 172
2268 204 198
2268 92 252
2268 2524 98
2268 2264 112
2268 3300 39
2268 3916 87
2269 2141 131
2269 3804 12
2269 2249 98
2269 69 187
2269 181 206
2269 2701 216
2269 2452 147
2269 604 19
2269 3285 235
2269 220 25
2270 590 49
2270 2780 248
2270 2202 99
2270 2077 37
2270 254 137
2270 724 80
2270 2302 178
2270 2262 198
2271 2207 219
2271 2773 53
2271 2261 77
2271 2303 173
2271 2303 202
2271 2537 168
2271 2525 133
2272 2208 178
2272 2528 159
2272 2549 134
2272 2528 229
2272 1125 50
2272 3296 209
2272 2049 251
2272 3296 165
2272 2148 68
2272 104 36
2272 2244 8
2272 3240 60
2272 3184 213
2272 240 241
2272 164 173
2272 2274 1
2272 2180 20
2272 2552 81
2272 2288 118
2272 2082 2
2272 480 136
2272 2275 176
2272 2532 27
2273 2276 112
2273 2721 163
2273 2124 135
2273 2497 196
2273 2400 56
2273 192 174
2273 2785 44
2273 353 162
2273 2785 130
2273 2255 75
2273 2277 119
2273 3121 165
2273 2281 4
2274 2055 102
2274 2798 23
2274 2290 140
2274 3758 179
2274 2528 248
2274 194 211
2274 3282 31
2274 2300 72
2274 2233 46
2274 2786 14
2274 2214 154
2274 2272 1
2274 245 210
2274 2144 187
2274 202 87
2274 1184 217
2274 2294 140
2275 1056 187
2275 2406 160
2275 2147 251
2275 131 55
2275 323 238
2275 2272 176
2275 67 165
2275 2123 252
2275 747 18
2275 2211 144
2275 2242 215
2275 1218 152
2275 3234 88
2276 2214 59
2276 2080 171
2276 2287 88
2276 3237 119
2276 2796 197
2276 401 99
2276 2176 22
2276 196 212
2276 3301 130
2276 1188 122
2276 2690 162
2276 2404 221
2276 3188 92
2276 744 122
2276 2210 140
2276 709 161
2276 2279 91
2276 1264 212
2276 3297 183
2276 2273 112
2276 2438 159
2276 2289 178
2276 2246 83
2276 133 88
2276 2148 89
2276 1262 238
2276 2284 5
2276 2146 100
2276 2288 187
2277 3239 121
2277 236 65
2277 3110 81
2277 2294 109
2277 2210 125
2277 2273 119
2277 3269 243
2277 3300 37
2278 102 246
2278 3762 138
2278 2656 37
2278 2160 168
2278 3318 138
2278 2374 195
2278 2790 203
2278 2146 227
2278 2214 243
2278 2948 92
2278 2530 10
2278 2284 97
2278 2166 27
2279 3321 4
2279 2213 220
2279 2276 91
2279 1254 189
2279 4069 9
2279 2165 164
2280 520 83
2280 3118 8
2280 2088 219
2280 3262 169
2280 2216 189
2280 2092 210
2280 2664 62
2280 2152 22
2280 2240 227
2280 3168 178
2280 200 157
2280 2105 71
2280 2216 227
2280 3745 75
2280 2249 249
2280 3304 182
2280 210 224
2280 3274 89
2281 2665 240
2281 3296 194
2281 745 201
2281 2241 83
2281 3176 168
2281 105 109
2281 2273 4
2281 2153 39
2281 1265 114
2281 3299 239
2282 3049 157
2282 2218 18
2282 238 62
2282 0 67
2282 712 72
2282 2080 74
2282 2722 29
2282 2218 102
2282 1114 210
2283 1257 189
2283 47 91
2283 1263 13
2283 218 76
2283 2539 14
2283 1321 160
2283 233 227
2283 2540 23
2283 243 220
2283 1251 205
2284 2157 141
2284 2684 112
2284 2244 141
2284 2276 5
2284 2285 128
2284 477 52
2284 2800 52
2284 3308 15
2284 228 231
2284 236 45
2284 2252 75
2284 2278 97
2284 3308 81
2284 192 7
2285 2157 181
2285 2284 128
2286 238 250
2286 2735 245
2286 1256 175
2286 2298 243
2286 2298 231
2286 3308 71
2286 2244 207
2286 428 162
2286 3534 194
2286 3310 88
2287 2798 53
2287 2123 211
2287 2219 23
2287 2159 125
2287 3278 242
2287 3299 213
2287 2276 88
2288 244 30
2288 2322 157
2288 2544 26
2288 2224 25
2288 2276 187
2288 3312 141
2288 2643 210
2288 2129 50
2288 2156 214
2288 2544 179
2288 2513 8
2288 2258 179
2288 2272 118
2288 2292 182
2288 112 212
2289 3313 206
2289 3811 199
2289 2276 178
2289 2532 163
2289 229 61
2289 3377 156
2289 2210 15
2289 2225 87
2289 2737 238
2289 240 148
2289 1056 73
2289 2165 45
2289 2259 18
2290 185 47
2290 2786 229
2290 2226 17
2290 288 163
2290 176 38
2290 2274 140
2290 2707 6
2290 3312 140
2290 2298 104
2290 1170 224
2290 2296 71
2290 2262 188
2290 2296 234
2290 130 78
2290 176 207
2290 2291 65
2291 2227 8
2291 2482 90
2291 2166 21
2291 115 85
2291 2258 188
2291 211 139
2291 434 16
2291 2198 101
2291 627 111
2291 2290 65
2291 2178 186
2291 2144 155
2292 508 126
2292 2295 167
2292 2294 162
2292 255 92
2292 2215 24
2292 2548 163
2292 2068 240
2292 2300 74
2292 2294 127
2292 2288 182
2292 464 250
2292 388 52
2292 228 191
2293 2789 22
2293 2107 201
2293 212 168
2293 3269 225
2293 2301 58
2293 2549 222
2294 3258 159
2294 3198 155
2294 2934 42
2294 2224 234
2294 3798 8
2294 100 245
2294 2438 1
2294 3222 177
2294 3574 196
2294 2292 162
2294 2215 117
2294 2274 140
2294 2486 187
2294 2277 109
2294 2164 100
2294 2742 176
2294 2085 172
2294 246 53
2294 2292 127
2295 2231 162
2295 114 157
2295 2292 167
2295 3318 234
2295 2239 5
2296 248 161
2296 530 12
2296 2233 166
2296 252 219
2296 2812 128
2296 2240 28
2296 2290 71
2296 2290 234
2296 3321 197
2296 2090 238
2296 152 120
2296 216 110
2297 3771 135
2297 2153 108
2297 2748 119
2297 2168 10
2297 698 116
2297 89 17
2297 3249 104
2297 752 66
2298 2746 219
2298 2286 243
2298 2286 231
2298 250 13
2298 2290 104
2298 2750 0
2298 2682 151
2298 2222 166
2299 3127 182
2299 243 160
2299 2462 4
2299 2185 229
2299 2553 172
2299 3771 255
2300 2274 72
2300 2292 74
2300 2132 144
2300 2244 102
2300 236 176
2300 1244 99
2300 230 153
2300 2812 252
2300 2302 135
2301 105 153
2301 2293 58
2301 489 112
2301 255 186
2302 2270 178
2302 502 170
2302 3325 79
2302 2662 61
2302 2300 135
2302 2334 18
2302 2110 134
2302 188 55
2302 252 158
2302 3802 197
2303 2525 174
2303 250 119
2303 2271 173
2303 3289 106
2303 3305 243
2303 2271 202
2303 245 173
2304 2336 91
2304 3712 61
2304 2595 204
2304 3848 40
2304 2314 19
2304 2176 8
2304 2452 91
2304 776 180
2304 2816 31
2304 2312 71
2304 2336 177
2304 2305 67
2304 484 22
2304 384 35
2304 320 217
2304 2305 96
2304 2048 52
2304 3328 44
2304 2370 30
2304 1028 16
2304 2340 34
2304 3137 171
2304 2816 81
2304 2826 45
2304 2817 93
2304 1418 6
2304 3848 115
2304 2376 138
2304 2832 248
2304 2066 20
2304 2320 198
2304 2848 101
2304 3360 89
2304 2313 3
2304 2208 83
2304 3328 248
2304 2408 120
2304 64 121
2304 2444 33
2304 342 177
2304 2500 220
2304 3364 186
2304 2312 49
2304 2817 196
2304 2408 125
2304 2116 72
2304 2176 206
2304 2048 1
2304 329 241
2304 3328 168
2304 256 36
2304 2306 82
2304 256 174
2304 272 199
2304 2432 134
2304 256 199
2305 3077 37
2305 2304 67
2305 2433 187
2305 2379 85
2305 2304 96
2305 2313 30
2305 2373 140
2305 2861 124
2305 2417 61
2305 3329 141
2305 257 167
2305 3969 153
2305 2313 108
2305 2096 101
2305 257 188
2305 257 111
2305 3873 127
2305 2864 98
2305 2816 8
2305 2323 156
2305 2307 54
2305 3365 53
2305 2321 131
2305 2953 40
2305 2137 209
2305 3913 145
2305 257 242
2305 272 44
2305 2817 13
2305 2440 206
2305 2065 91
2305 2593 218
2305 2310 203
2306 2312 220
2306 2050 116
2306 3120 152
2306 2362 165
2306 2308 10
2306 2312 212
2306 3370 81
2306 2386 23
2306 2310 8
2306 2434 21
2306 3091 203
2306 259 147
2306 2310 236
2306 512 12
2306 2182 124
2306 774 232
2306 3430 158
2306 3392 138
2306 146 175
2306 401 125
2306 10 75
2306 16 91
2306 3074 74
2306 1296 100
2306 2816 207
2306 2322 138
2306 1282 68
2306 2050 249
2306 2344 123
2306 2314 46
2306 2316 100
2306 2358 33
2306 2310 19
2306 2081 26
2306 2561 29
2306 2304 82
2307 3077 235
2307 2319 85
2307 2311 118
2307 275 29
2307 402 15
2307 2433 227
2307 3395 154
2307 1155 236
2307 2305 54
2307 2881 226
2307 261 242
2307 2339 244
2307 2321 234
2307 1283 153
2307 2054 118
2307 2178 217
2307 641 8
2307 2314 243
2307 2826 97
2307 2129 118
2307 2337 16
2307 1027 73
2307 2073 80
2307 337 137
2307 2341 83
2307 3331 208
2307 2339 75
2307 1378 111
2307 2062 235
2307 2699 57
2307 2437 200
2307 259 219
2307 2369 1
2307 3939 224
2308 2468 253
2308 2306 10
2308 2316 84
2308 2566 182
2308 256 27
2308 308 87
2308 2324 210
2308 3334 237
2308 0 52
2308 2486 93
2308 2316 174
2308 276 96
2308 2052 39
2308 392 7
2308 3845 163
2308 820 198
2308 1302 5
2308 2309 29
2308 2820 119
2308 260 133
2308 2341 212
2308 2370 204
2308 2176 1
2308 278 28
2308 265 22
2308 2816 170
2308 2328 111
2308 1795 151
2308 393 94
2308 3364 148
2308 2069 173
2308 260 0
2308 2380 149
2308 2340 189
2308 2048 219
2308 2836 213
2308 4001 231
2308 3332 72
2308 260 102
2308 1316 124
2308 780 8
2308 256 91
2308 308 41
2308 2837 182
2308 2436 90
2308 256 59
2308 2820 72
2308 2820 166
2309 1285 136
2309 2325 90
2309 2313 123
2309 2839 230
2309 3875 196
2309 2853 187
2309 2053 82
2309 2834 27
2309 2180 180
2309 2068 118
2309 2947 254
2309 2348 220
2309 3333 99
2309 2434 153
2309 1285 252
2309 2308 29
2309 2693 220
2309 2085 95
2309 385 109
2309 3073 122
2309 260 175
2309 2439 67
2309 2565 60
2309 261 74
2309 2560 188
2309 2311 216
2310 3990 8
2310 2386 145
2310 1284 132
2310 2438 103
2310 2306 19
2310 6 91
2310 2326 86
2310 2656 145
2310 2050 79
2310 771 74
2310 2407 206
2310 597 159
2310 52 117
2310 2244 112
2310 2306 8
2310 2306 236
2310 3343 177
2310 3074 224
2310 2319 104
2310 1302 254
2310 2316 237
2310 70 228
2310 2330 253
2310 324 74
2310 2438 158
2310 262 245
2310 2305 203
2310 2566 120
2310 2454 155
2310 2454 192
2310 2439 242
2310 2406 1
