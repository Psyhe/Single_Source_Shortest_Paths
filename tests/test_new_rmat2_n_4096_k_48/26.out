55
19
11
52
14
110
19
49
51
56
25
51
77
174
21
55
13
56
15
61
66
42
33
39
59
50
33
89
56
48
13
26
87
76
59
92
108
92
11
18
67
67
83
63
60
129
23
81
24
59
32
81
56
96
45
83
67
71
27
155
113
36
34
58
38
27
45
109
26
179
40
43
50
69
96
161
67
182
21
23
22
35
12
41
25
